File: Beginner/C Sharp/1012 - Area/1012 - Area.cs
using System;

namespace _1012___Area
{
    class Program
    {
        static void Main(string[] args)
        {

            //Declare Variables
            double pi = 3.14159;
            string values = Console.ReadLine();
            string[] values2 = values.Split(' ');
            double A = Convert.ToDouble(values2[0]);
            double B = Convert.ToDouble(values2[1]);
            double C = Convert.ToDouble(values2[2]);

            //Calculate
            double triangule = (A * C) / 2.0;
            double circle = Math.Pow(C, 2) * pi;
            double trapezio = (C * (A + B)) / 2;
            double square = Math.Pow(B, 2);
            double rectangle = A * B;

            //Print
            Console.WriteLine("TRIANGULO: {0:0.000}", triangule);
            Console.WriteLine("CIRCULO: {0:0.000}", circle);
            Console.WriteLine("TRAPEZIO: {0:0.000}", trapezio);
            Console.WriteLine("QUADRADO: {0:0.000}", square);
            Console.WriteLine("RETANGULO: {0:0.000}", rectangle);

        }
    }
}

File: Beginner/Python 3/1012 - Area.py
# Declare Variables
numbers = input().split(' ')
A = float(numbers[0])
B = float(numbers[1])
C = float(numbers[2])
pi = 3.14159

# Calculate Areas
triangle = (A * C) / 2
circle = C * C * pi
trapezium = C * ((A + B) / 2)
square = B * B
rectangle = A * B

# Show result
print("TRIANGULO: {:.3f}".format(triangle))
print("CIRCULO: {:.3f}".format(circle))
print("TRAPEZIO: {:.3f}".format(trapezium))
print("QUADRADO: {:.3f}".format(square))
print("RETANGULO: {:.3f}".format(rectangle))
